upsertGitHubTag/deployment/index.js

```diff
@@ -269,8 +269,6 @@ function processEvent(event, callback) {
         " from GitHub.",
     });
   }
-
-  callback(null, { statusCode: 200, body: "results" });
```

Comment:
Yeah, we probably should do something to prevent the subsequent code from being run.  The `else if` would be fine, but I'd be even more inclined to add a `return` at line 207.  That way we get the heck out of there, for sure.

Reply:
Good catch, but why doesn't the handleCallback at line 234 execute first and send an error rather than line 273 first?

Reply:
In a nutshell, the `handleCallback` at line 235 (I added the `return` and bumped it down a line) is in the body of a callback function which is called in a function that is installed as a handler for the `end` event of the http request, 
https://github.com/dockstore/lambda/blob/2532890d5d372fffd99a84abaad9ce7ba6e0b763/upsertGitHubTag/deployment/index.js#L58
So, in that case, the lambda callback is invoked after the http response has been received.